<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateinfo</class>
 <widget class="QMainWindow" name="updateinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 21, 36);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid #8f8f91;
    border-radius: 15px;
    margin-top: 10px; 
    background-color:rgb(255, 255, 255,0);
    padding: 5px;
}</string>
    </property>
    <property name="title">
     <string>UPDATE INFO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_name">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    border: 2px solid #8f8f91;
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Phone No.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_phno1">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    border: 2px solid #8f8f91;
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_create_email">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    border: 2px solid #8f8f91;
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_create_password">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    border: 2px solid #8f8f91;
    border-radius: 10px; 
    background-color: rgb(255, 255, 255,0);
    min-width: 8px; 
    min-height: 8px;
    padding: 5px;
    margin: 0px;
}
</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 10px; 
    background-color:rgb(255, 255, 255,0);
    min-width: 10px; 
    min-height: 10px; 
    padding: 5px; 
    margin: 0px; 
}
</string>
       </property>
       <property name="text">
        <string>UPDATE INFO</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoHome"/>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
